qemu/tests/cfg/fio_perf.cfg

```diff
@@ -101,6 +101,7 @@
     variants:
         - single_disk:
             no virtio_fs_perf
+            no aarch64
```

Comment:
Actually, I don't think there is a need for restrictions in the code here.
Because the aarch64 platform is also supported here.
We can use '--testcase=fio_perf.shared_fs' to control.

Reply:
> Actually, I don't think there is a need for restrictions in the code here. Because the aarch64 platform is also supported here. We can use '--testcase=fio_perf.shared_fs' to control.

Done.